### DISCUSSION TITLE
Plasticity parameters

### DISCUSSION THREAD

**Discussion by sajutabraham** [neperfepx/FEPX#41] [Q&A] [answered]
Could anyone suggest why the term `d0` is given with the parameters like m, gammadot_0, h_0, g_0, gs_0 and n? It does not appear for the elastic constants C11, C12 and C44. I am talking based on the `simulation_config` file. Also, where can I get more details of these parameters? Thanks in advance.

**Comment by mkasemer** [comment, marked as answer]
Hi @sajutabraham.

The value of `d0` is appended to all values that are not given in exponential form in order to preserve precision. In some instances, leaving values without `d0` leads to performance issues in FEPX. This is a known bug with no timeline for a fix. Until then, please give `d0` on all values not in exponential form. Note that the other values, like the elastic constants, have an exponent.

Explanations for these material parameters can either be found in the manual, or in any number of the journal articles listed on the [applications page](https://fepx.info/applications.html) of the FEPX website.

**Reply by smallrain1996** [reply]
It seems to be a very important issue, in previous versions (e.g. 1.2.1) there was no d0 after n and m, whereas in the latest version (1.2.2-1) there is now a d0 after both n and m.

**Reply by acmelab-ua** [reply]
Yes, it has the potential to introduce error into the simulations. Future versions of the code will act very cautiously and add `d0` to every non-exponential value, and `dn` to every exponential value (where `n` is the exponent), until the issue is fully understood and resolved.

The error is not persistent, and does not affect all values.
